xml/System.Net.Http/HttpClient.xml

```diff
@@ -426,11 +426,14 @@ The default instance returned by this property will initialize following a diffe
 * **For Linux:** Reads proxy configuration from environment variables or, in case those are not defined, this property initializes a non-configured instance that bypasses all addresses.
 
 The environment variables used for `DefaultProxy` initialization on Windows and Unix-based platforms are:
-* HTTP_PROXY: the hostname or IP address of the proxy server used on HTTP requests.
-* HTTPS_PROXY: the hostname or IP address of the proxy server used on HTTPS requests.
-* ALL_PROXY: the hostname or IP address of the proxy server used on HTTP and/or HTTPS requests in case HTTP_PROXY and/or HTTPS_PROXY are not defined.
+* HTTP_PROXY: the proxy server used on HTTP requests.
+* HTTPS_PROXY: the proxy server used on HTTPS requests.
+* ALL_PROXY: the proxy server used on HTTP and/or HTTPS requests in case HTTP_PROXY and/or HTTPS_PROXY are not defined.
 * NO_PROXY: a comma-separated list of hostnames that should be excluded from proxying.
 
+On systems where environment variables are case-sensitive, the variable names may be all lower-case or all upper-case.  The all lower-case names are checked first.
+
+The proxy server may be a hostname or IP address, optionally followed by a colon and port number, or it may be an http URL, optionally including a username and password for proxy authentication.  The URL must be http (https is not currently supported) and must not include any text after the hostname, IP, or port.
```

Comment:
```suggestion
The proxy server may be a hostname or IP address, optionally followed by a colon and port number, or it may be an http URL, optionally including a username and password for proxy authentication. The URL must be start with `http`, not `https`, and cannot include any text after the hostname, IP, or port.
```